Tests/KarrotFlexTests/KarrotFlexTestView.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  KarrotFlexTestView.swift
+//  
+//
+//  Created by Kanghoon Oh on 2022/01/14.
```

Comment:
I think, KarrotFlex seems to need copyright convention.

Reply:
I think it's look so good. how about you? @magi82

Reply:
I think I'm good too.

Reply:
I modified it here. 56044e5